bin/compile.dart

```diff
@@ -0,0 +1,120 @@
+//  Copyright 2021 Abitofevrything and others.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:logging/logging.dart';
+
+import 'compile/generator.dart';
+
+void main(List<String> args) async {
+  late ArgParser parser;
+  parser = ArgParser()
+    ..addFlag(
+      'help',
+      abbr: 'h',
+      negatable: false,
+      help: 'Print this help and exit',
+    )
+    ..addOption(
+      'output',
+      abbr: 'o',
+      defaultsTo: 'out.g.dart',
+      help: 'The file where generated output should be written to',
+    )
+    ..addOption(
+      'verbosity',
+      abbr: 'v',
+      defaultsTo: 'info',
+      allowed: Level.LEVELS.map((e) => e.name.toLowerCase()),
+      help: 'Change the verbosity level of the command-line output',
```

Comment:
please add all possible options in help, or they already listed if set in allowed?

Reply:
The options are already listed in the help generated for each option by `package:args`